packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.ts

```diff
@@ -100,8 +100,10 @@ describe("End to end chunked encoding", () => {
 			);
 			return getTreeContext(
 				schema,
-				editableForest,
-				dummyEditor,
+				{
+					forest: editableForest,
+					editor: dummyEditor,
+				} as unknown as ITreeCheckout,
```

Comment:
Casting as `ITreeCheckout` is not quite correct because the editor we're passing in is an `IDefaultEditBuilder` not an `ISharedTreeEditor`. The tests don't use the schema edition capabilities in the latter, so they still pass. Happy to hear suggestions of how else could we pass this to make it cleaner.